modules/freewheelSSPBidAdapter.js

```diff
@@ -223,6 +223,24 @@ export const spec = {
       componentId: getComponentId(currentBidRequest.params.format)
     };
 
+    // Add GDPR flag and consent string
+    if (bidderRequest.gdprConsent) {
+      requestParams._fw_gdpr_consent = bidderRequest.gdprConsent.consentString;
+
+      if (typeof currentBidRequest.gdprConsent.gdprApplies === 'boolean') {
+        requestParams._fw_gdpr = currentBidRequest.gdprConsent.gdprApplies;
```

Comment:
Should update `currentBidRequest` to `bidderRequest` in these two lines of code.

Reply:
done.